app/services/application/ApplicationService.scala

```diff
@@ -560,10 +567,11 @@ trait ApplicationService extends EventSink with CurrentSchemeStatusHelper {
       _ <- reviewerAssessmentScoresRepository.resetExercise(UniqueIdentifier(applicationId), reviewerExercisesToRemove)
       _ <- fsacRepo.removeFsacEvaluation(applicationId)
       _ <- rollbackAppAndProgressStatus(applicationId, ApplicationStatus.ASSESSMENT_CENTRE, statuses)
-      // Read back the video results and set the current scheme status to these
-      phase3ResultsOpt <- getPhase3Results(applicationId)
+      phase3ResultsOpt <- getPhase3Results
       phase3Results = phase3ResultsOpt.getOrElse(throw new RuntimeException("No phase 3 video results found"))
-      _ <- appRepository.updateCurrentSchemeStatus(applicationId, phase3Results)
+      siftResults <- fetchSiftResults
+      css = calculateCurrentSchemeStatus(phase3Results, siftResults)
+      _ <- appRepository.updateCurrentSchemeStatus(applicationId, css)
```

Comment:
how current scheme status will looks like if there was no sifting scheme? 
Shouldn't it be join of css + phase3Results?